### ISSUE TITLE
Eigfact for SymmetricRFP

### ISSUE THREAD

**Issue by mfalt** [andreasnoack/LinearAlgebra.jl#18]
I am working on a project where I wanted to do eigenvalue decomposition on Rectangular Full Packed Matrices and found the [sspev](http://www.netlib.org/lapack/explore-html/d5/d16/sspev_8f_aee30ff1f2a1f001a8c75a1d233985bb5.html#aee30ff1f2a1f001a8c75a1d233985bb5)/[dspev](http://www.netlib.org/lapack/explore-html/d4/d0f/dspev_8f_aa31d35b52c937fdbacaa2e034d8bb52b.html#aa31d35b52c937fdbacaa2e034d8bb52b) functions in LAPACK. Are you still actively working on this package and would you be interested in a PR adding
`eig` and `spev` to `rectfullpacked.jl` and `lapack.jl`?

**Comment by andreasnoack** [comment]
Yes, definitely. It's been a long time since I've been looking at the RFP code and it might need some updating. Adding eigen factorizations for the RFP would be great so please prepare a PR.

**Comment by mfalt** [comment]
It seems to me that a lot of code in this package is broken, with missing imports and similar so I will not spend the time needed to get this up and working. As a reference of some of the things needed to just create a SymmetricRFP, print it and call cholfact (still doesn't seem to produce the right answers) you can look at my branch https://github.com/mfalt/LinearAlgebra.jl but I will settle with only implementing the bare minimum in my own package, you may close this issue if you want. As a reference, I'm running julia 0.5.

**Comment by andreasnoack** [comment]
I've taken a look at it. Most of the fixes were straight forward. At least when you are used to Julia's linear algebra code but the `getindex` was little tedious to get right. See #19 where I've also added some tests. Hopefully, the code is now in a state where you can start working on it. Alternative, I'll probably wrap the eigensolvers at some point.
